skeleton/src/main/java/com/dropbox/kaiken/scoping/SingleIn.kt

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
this is the definition of our app,user and activity components. We define the subcomponent specs here and anvil wires them to the skeleton scope we create within each application.